SteakBot.Core/Modules/HelpModule.cs

```diff
@@ -23,11 +24,18 @@ public async Task List()
         [Command("help")]
         public async Task Help()
         {
-            await ReplyAsync("`hi` - Bot says \"Hi\"");
-            await ReplyAsync("`say <something>` - Bot says something");
-            await ReplyAsync("`list` - Lists all available \"memes\"");
-            await ReplyAsync("`quote` - Duuh, quotes...");
-            await ReplyAsync("`addMeme <commandName> <value(link/ascii art/etc)> <description(use \"\" to format longer descriptions)>` - Add a dank meme of your choosing");
+            StringBuilder helpStringBuilder = new StringBuilder();
+
+            helpStringBuilder.AppendLine("`hi` - Bot says \"Hi\"");
+            helpStringBuilder.AppendLine("`say <something>` - Bot says something");
+            helpStringBuilder.AppendLine("`list` - Lists all available \"memes\"");
+            helpStringBuilder.AppendLine("`quote` - Duuh, quotes...");
+            helpStringBuilder.AppendLine("`addMeme <commandName> <value(text/link)> <description(use \"\" when you have spaces)>` - Add a dank meme of your choosing");
+            helpStringBuilder.AppendLine("`editMem <commandName> <newCommandName> <newValue(text/link)> <newDescription(use \"\" when you have spaces)>` - Evolve a meme of your choosing");
+            helpStringBuilder.AppendLine("`removeMeme <commandName>` - Remove an ancient meme.");
+            helpStringBuilder.AppendLine("Meme type parameters are the following: `Text, Image, Video`.");
+
+            await ReplyAsync(helpStringBuilder.ToString());
```

Comment:
Since the result of this string builder will always be the same it can be moved to the constructor and assigned to a private readonly field and reused.

Reply:
Corrected.